<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="playerWindow"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="playButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pauseButton">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rewButton">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>rew</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="forwardButton">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ff</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="seekPosition">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>651</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>681</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>654</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>656</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>680</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rewButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>rewind()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>655</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forwardButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>656</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>682</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>seekPosition</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>seek(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>654</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>678</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>play()</slot>
  <slot>pause()</slot>
  <slot>stop()</slot>
  <slot>rewind()</slot>
  <slot>forward()</slot>
  <slot>seek(int)</slot>
 </slots>
</ui>
